app/src/main/res/values/strings.xml

```diff
@@ -998,9 +998,10 @@ A level counts as a roof level when its windows are in the roof. Subsequently, r
     <string name="quest_crossing_kerb_height_title">What’s the height of the curbs at this crossing?</string>
 
     <string name="quest_crossing_title2">Is there a crossing here?</string>
-    <string name="quest_crossing_yes">Yes (e.g. the curb is lowered here or there are markings)</string>
-    <string name="quest_crossing_no">No, but crossing is possible</string>
+    <string name="quest_crossing_yes">Yes, intended as crossing</string>
```

Comment:
I think the examples were really helpful. Can we change this to "Yes, intended as crossing (e.g. with lowered curb or markings)"?

Reply:
I gave the reason for removing it in https://github.com/streetcomplete/StreetComplete/pull/5936#discussion_r1785984014 . Do you disagree?

Reply:
Ah, I haven't seen the explanation before. So it should be fine.